src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Quadrangle {
+    public IsoscelesTrapezoid(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();
```

Comment:
You should not simply return the `super.getArea()` method, as this does not calculate the specific area for an isosceles trapezoid. You need to implement the area calculation logic specific to an isosceles trapezoid.